python/create_mask.py

```diff
@@ -0,0 +1,73 @@
+from datetime import datetime
```

Comment:
Yay!!! 1st python file in SolarMonitor!!! thanks @GartontT!!! Though now solarmonitor has one more language in the bucket!